cli/src/commands/auth.rs

```diff
@@ -0,0 +1,46 @@
+use crate::commands::Command;
+use crate::config::CliConfig;
+use crate::errors::{ExitCode, Fallible};
+use crate::telemetry::Session;
+use crate::terminal::{confirm, input};
+use log::{info, warn};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub enum Auth {
+    /// Setup your auth stuff
+    /// Requires using an User key which can be found here:
+    /// https://engine.apollographql.com/user-settings
+    Setup(Setup),
+}
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Setup {}
+
+impl Command for Setup {
+    fn run(&self, _session: &mut Session) -> Fallible<ExitCode> {
+        let mut config = CliConfig::load().unwrap();
```

Comment:
I don't think we should `.unwrap()` here and instead we should handle the error with some dedicated output of what went wrong